Mathlib/MeasureTheory/OuterMeasure/AE.lean

```diff
@@ -32,18 +32,29 @@ However, we restate some lemmas specifically for `ae`.
 outer measure, measure, almost everywhere
 -/
 
-open Filter Set
+open Filter Set MeasureTheory
 open scoped ENNReal
 
-namespace MeasureTheory
-
 variable {α β F : Type*} [FunLike F (Set α) ℝ≥0∞] [OuterMeasureClass F α] {μ : F} {s t : Set α}
 
 /-- The “almost everywhere” filter of co-null sets. -/
-def ae (μ : F) : Filter α :=
+def MeasureTheory.ae (μ : F) : Filter α :=
   .ofCountableUnion (μ · = 0) (fun _S hSc ↦ (measure_sUnion_null_iff hSc).2) fun _t ht _s hs ↦
     measure_mono_null hs ht
 
+namespace Mathlib.Meta
+open Lean Elab Term Meta Qq
+
+/-- Try to elaborate `μ` as a term of type `T` where `OuterMeasure T`. If that fails, try to
+elaborate `μ` as `Measure ?Ω`. -/
```

Comment:
You may want to try `Measure ?_` first, because otherwise Lean may decide to cast a `Measure` to an `OuterMeasure`.

Reply:
Sorry, you got misled by a typo in my docstring. It's supposed to be
```suggestion
/-- Try to elaborate `μ` as a term of type `T` where `OuterMeasureClass T`. If that fails, try to
elaborate `μ` as `Measure ?Ω`. -/
```